src/PrestaShopBundle/Twig/TranslationsExtension.php

```diff
@@ -115,11 +129,14 @@ public function concatenateEditTranslationForm($subtree, $viewProperties)
      * Returns a tree of translations key values.
      *
      * @param array $translationsTree
+     * @param null $theme
      *
      * @return string
      */
-    public function getTranslationsTree(array $translationsTree)
+    public function getTranslationsTree(array $translationsTree, $theme = null)
```

Comment:
ok, I think I can be more easy with theme as a string **if** we call it ``$themeName``: WDYT ?